apps/project/models.py

```diff
@@ -176,6 +179,32 @@ def get_recent_activities(cls, user):
             for item in leads_qs.union(entry_qs).union(entry_comment_qs).order_by('-created_at')[:30]
         ]
 
+    @staticmethod
+    def get_change_attribute_project(user):
```

Comment:
Let's change the function name to get_recent_active_projects. And do all the generating project queryset logic here as well.